boards/arduino-atmega-common/include/arduino_pinmap.h

```diff
@@ -32,6 +34,40 @@ extern "C" {
  *
  * @note    ISCP pins are not mapped.
  */
+
+/* Digital pins */
+#ifdef CPU_ATMEGA328P
```

Comment:
Where does this define come from ?
Maybe better use avr-lib \_\_AVR_ATmega328P\_\_ defines ?